index.js

```diff
@@ -18,10 +18,7 @@ const loadPlugin = (plugin, ...args) => {
 	}
 };
 
-const exposePlugin = plugin =>
-	function (...args) {
-		return loadPlugin(plugin, args);
-	};
+const exposePlugin = plugin => (...args) => loadPlugin(plugin, args);
```

Comment:
Hmm this doesn't seem to work @sindresorhus either

Reply:
My change here is correct. See: https://github.com/sindresorhus/gulp-imagemin/commit/c7170ba2cd03f888e30e79dbbc52ee7983d99300

Reply:
I'm still getting wrong results with 5.0.2 in a repo of mine where I pass extra options

```js
imageoptim.svgo.js2svg = {
    indent: 2,
    pretty: true
  };
```

Then I still get the SVGs not beautified